test/configs/backstop_features.js

```diff
@@ -121,6 +121,12 @@ module.exports = {
       selectors: ['.moneyshot'],
       misMatchThreshold: 5.0,
       requireSameDimensions: false
+    },
+    {
+      label: 'scrollTo',
+      url: '../../index.html',
+      scrollToSelector: '.puppy-link',
+      selectors: ['puppy-link']
```

Comment:
I thought this feature was intended to test a child-element inside a scrollable div?   I thought the below-the-fold elements that were direct children of the body tag already worked correctly?   Am I missing something here?

Reply:
It is... how can I write a backstop_features test that can imitate this? Do I need to add a scrollable div to index.html?

Reply:
In this case — I think it’s best that you create a folder in the test directory, maybe call it “special_cases” or something like that.  And in there you can add an html file which easily simulates the issue.  For that scenario you can point to that file.  Does that make sense?

Reply:
I just thought — what might be better instead is to add a folder to the examples directory and call it “scrolling_cases”.  In there you can add an html file which simulated the issue and a backstop config which demonstrates how to test the case.  You can add a short readme with a sentence which explains the issue and how to resolve it so other users can learn from your example.

Reply:
ohhhh gotcha! OK I will work on adding that